public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
- [ ] All boilerplate files and icons should be deleted as they are not required for the project. e.g In the public folder(the logo images, manifest.json and robots.txt).